dbt_project.yml

```diff
@@ -23,4 +23,5 @@ on-run-start:
   - "{{ elementary.create_elementary_tests_schema() if flags.WHICH in ['test', 'build'] }}"
 on-run-end:
   - "{{ elementary.handle_tests_results(results) if flags.WHICH in ['test', 'build'] }}"
-  - "{{ elementary.upload_run_results(results) }}"
+  - "{{ elementary.upload_run_results(results) if not elementary.get_config_var('disable_run_results') }}"
+  - "{{ elementary.upload_dbt_artifacts() if not elementary.get_config_var('disable_uploading_dbt_artifacts') }}"
```

Comment:
What do you think about doing this first? then if for some reason the code in upload_run_results and handle_test_results is relying on fresh metadata it will work